code_to_optimize/bubble_sort_method.py

```diff
@@ -3,6 +3,7 @@ def __init__(self, x=0):
         self.x = x
 
     def sorter(self, arr):
+        print("codeflash stdout : BubbleSorter.sorter() called")
```

Comment:
for this example, can you also add a stderr print statement? I want to ensure that we dont capture stderr